### DISCUSSION TITLE
Install on a Mac?

### DISCUSSION THREAD

**Discussion by fpolli** [coqui-ai/TTS#1448] [General Q&A] [answered]
Hi, I apologize if this is uber basic or has been answered, but I did run a couple searches here and on StackOverflow, and I did not see anything on installing on a Mac. I have an iMac running Monterey. I only see instructions for Ubuntu here and a SO post for Windows. Are there any steps different than the Ubuntu instructions for a Mac?

Thank you

**Comment by erogol** [comment, marked as answer]
to make it work on Mac, we need to wait for https://github.com/pytorch/pytorch/issues/47702

I made it work by installing everything with conda but it is not optimal

**Reply by fpolli** [reply]
Thank you. My iMac is a few years old, so it is an Intel.

I will explore conda. Thanks!
